test/parser/test.ml

```diff
@@ -467,43 +467,39 @@ let tests : test_suite list = [
     t "preceded-by-shorthand-list" "- foo\n{2 Bar}";
     t "nested-in-two-lists" "{ul {li - foo\n{2 Bar}}}";
     t "bad-level-long-number" "{22 Foo}";
-    t "bad-level-title" "{1 Foo}";
-    t "bad-level-too-deep" "{5 Foo}";
+    t "bad-level-title" "{0 Foo}";
+    t "bad-level-too-deep" "{6 Foo}";
     t "link-in-markup" "{2 {{:foo}}}";
     t "reference-in-markup" "{2 {!foo}}";
   ];
 
   "section-contexts", [
-    t "titles-allowed" "{1 Foo}"
+    t "titles-allowed" "{0 Foo}"
       ~sections_allowed:`All;
-    t "titles-zero-not-allowed" "{0 Foo}"
```

Comment:
This test was deleted, so the expect file should be deleted, too. Perhaps, in a separate PR, we should have the tester scan for unused expect files.